onnxruntime/core/flatbuffers/ort.fbs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace onnxruntime.Attribute;
```

Comment:
Could maybe use 'onnxruntime.experimental' instead of just 'onnxruntime' as the top level namespace so we clearly advertise this is subject to change short term.

Reply:
Maybe 'onnxruntime.experimental.fbs."? 

---
In reply to: [473617546](https://github.com/microsoft/onnxruntime/pull/4870#discussion_r473617546) [](ancestors = 473617546)